### DISCUSSION TITLE
Installation doesn't seem to show v22 after it's run

### DISCUSSION THREAD

**Discussion by Fayyaadh** [nzbgetcom/nzbget#149] [General]
Hi there

So I followed the instructions [here](https://github.com/nzbgetcom/nzbget/discussions/100#discussioncomment-8080102) to update from v21 to v22.

The .run file runs and then I can access the webgui again.

However it still shows as v21 but if I got to Settings -> System -> Check for updates, it shows that v22 is available and if I click install, it fails with this error:

```
Executing update-script /root/nzbget/install-update.sh
Downloading version information...
Download failed, please try again later
```

**Comment by Fayyaadh** [comment]
Sorted.

For anyone else having the same issue, the instructions linked above didn't mention that you need to stop the nzbget service.

Once I did a `systemctl restart nzbget` it showed as version 22.

**Comment by phnzb** [comment]
@Fayyaadh Thanks for the feedback! Instruction updated.
